src/js/common/schemaform/reducers/index.js

```diff
@@ -147,35 +153,61 @@ export default function createSchemaFormReducer(formConfig) {
       }
       case SET_SAVE_FORM_STATUS: {
         const newState = _.set('savedStatus', action.status, state);
+        newState.startingOver = false;
+        newState.prefillStatus = PREFILL_STATUSES.notAttempted;
 
         // This is the only time we have a saved datetime
         if (action.status === SAVE_STATUSES.success) {
-          return _.set('lastSavedDate', action.lastSavedDate, newState);
+          newState.lastSavedDate = action.lastSavedDate;
         }
 
         return newState;
       }
       case SET_FETCH_FORM_STATUS: {
         return _.set('loadedStatus', action.status, state);
       }
+      case SET_FETCH_FORM_PENDING: {
+        const newState = _.set('loadedStatus', LOAD_STATUSES.pending, state);
+
+        if (action.prefill) {
+          newState.prefillStatus = PREFILL_STATUSES.pending;
```

Comment:
I think that makes the form status action confusing, since it's directly specifying that we're changing the form status.

This is something I don't like about how we use Redux actions here (which is much my fault as anyone's): they're often focused on changing a specific piece of state instead of relating to an actual event that has happened. What really happened here is we started the process of fetching a form; the changing form status is a consequence of that. I probably should name this something like `START_FETCH_FORM`, but that felt like straying too far from the existing Redux actions.